### PR TITLE
Add note about conditional man pages

### PR DESCRIPTION
**Pull request by jubalh**
Closes https://github.com/shadow-maint/shadow/issues/83

### CODE REVIEW

File: man/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The [upstream tarball](https://github.com/shadow-maint/shadow/releases) ships
+with pre-built manpages.
+
+The content of the man pages however is dependent on compile flags. So the pre-built ones might not fit your version of shadow.
```
Comment (vapier) [inline man/README.md]: wrap to 80 cols ?

File: man/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+The [upstream tarball](https://github.com/shadow-maint/shadow/releases) ships
```
Comment (vapier) [inline man/README.md]: s/upstream tarball/official releases/

**Comment by jubalh** [comment]
@vapier thanks for the review

**Comment by jubalh** [comment]
@hallyn what about this one?

**Comment by hallyn** [comment]
Thanks.

**Pull request merged**